acp/identity/identity.go

```diff
@@ -34,33 +42,95 @@ type Identity struct {
 	// The address is derived from the actor's public key,
 	// using the did:key method
 	DID string
+
+	BearerToken string
 }
 
 // FromPrivateKey returns a new identity using the given private key.
-func FromPrivateKey(privateKey *secp256k1.PrivateKey) (immutable.Option[Identity], error) {
-	pubKey := privateKey.PubKey()
-	did, err := DIDFromPublicKey(pubKey)
+//
+//   - duration: The [time.Duration] that this identity is valid for.
+//   - audience: The audience that this identity is valid for.  This is required
+//     by the Defra http client.  For example `github.com/sourcenetwork/defradb`
+//   - authorizedAccount: An account that this identity is authorizing to make
+//     SourceHub calls on behalf of this actor.  This is currently required when
+//     using the SourceHub ACP module.
+func FromPrivateKey(
+	privateKey *secp256k1.PrivateKey,
+	duration time.Duration,
+	audience immutable.Option[string],
+	authorizedAccount immutable.Option[string],
+) (Identity, error) {
+	publicKey := privateKey.PubKey()
+	did, err := DIDFromPublicKey(publicKey)
+	if err != nil {
+		return Identity{}, err
+	}
+
+	subject := hex.EncodeToString(publicKey.SerializeCompressed())
+	now := time.Now()
```

Comment:
It should not require it - `time.Time` is timezone agnostic until scoping it to one I think.  And when it gets serialized it will include the timezone component (meaning it will resolve correctly upon receive by another sensible endpoint).